validator/client/validator.go

```diff
@@ -338,12 +350,24 @@ func (v *validator) UpdateDuties(ctx context.Context, slot uint64) error {
 	if slot%params.BeaconConfig().SlotsPerEpoch == 0 || firstDutiesReceived {
 		for _, duty := range dutiesNextEpoch.Duties {
 			if duty.Status == ethpb.ValidatorStatus_ACTIVE {
-				aggregator, err := v.isAggregator(ctx, duty.Committee, duty.AttesterSlot, bytesutil.ToBytes48(duty.PublicKey))
+				attesterSlot := duty.AttesterSlot
+				committeeIndex := duty.CommitteeIndex
+
+				alreadySubscribedKey := bytesutil.ToBytes64(append(bytesutil.Bytes32(attesterSlot), bytesutil.Bytes32(committeeIndex)...))
```

Comment:
Can this be a helper func? It's a bit long

Reply:
Yep. Done